doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-execution-rest-api-proc.adoc

```diff
@@ -0,0 +1,94 @@
+[id='test-scenarios-execution-rest-api-proc']
+
+= Executing a test scenario using the {KIE_SERVER} REST API
+
+Directly interacting with the REST endpoints of {KIE_SERVER} provides the most separation between the calling code and the decision logic definition.
+You can use the {KIE_SERVER} REST API to execute the test scenarios externally. It executes the test scenarios against the deployed project.
+
+[NOTE]
+====
+This functionality is disabled by default, use `org.kie.scenariosimulation.server.ext.disabled` system property to enable it.
+====
+
+For more information about the {KIE_SERVER} REST API, see
+ifdef::DM,PAM[]
+{URL_KIE_APIS}[_{KIE_APIS}_].
+endif::[]
+ifdef::DROOLS,JBPM,OP[]
+xref:kie-server-rest-api-con_kie-apis[].
+endif::[]
+
+.Prerequisites
+
+* {KIE_SERVER} is installed and configured, including a known user name and credentials for a user with the `kie-server` role. For installation options, see
+ifdef::DM,PAM[]
+{URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
+endif::[]
+ifdef::DROOLS,JBPM,OP[]
+<<_installationandsetup>>.
+endif::[]
+
+* You have built the test scenario project as a KJAR artifact and deployed it to {KIE_SERVER}.
+* You have the ID of the KIE container.
+
+.Procedure
+
+. Determine the base URL for accessing the {KIE_SERVER} REST API endpoints. This requires knowing the following values (with the default local deployment values as an example):
++
+--
+* Host (`localhost`)
+* Port (`8080`)
+* Root context (`kie-server`)
+* Base REST path (`services/rest/`)
+
```

Comment:
Shouldnt we add a parameters mentioning 
/server/containers/[containerId]/scesim"